packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -426,9 +424,10 @@ export default class AccountTransferStore {
 
             const obj_values: TAccount = {
                 text:
-                    is_cfd && account.account_type === CFD_PLATFORMS.MT5 && combined_cfd_mt5_account
-                        ? `${combined_cfd_mt5_account.sub_title}${short_code_and_region}`
-                        : account_text_display,
+                    is_cfd &&
+                    account.account_type === CFD_PLATFORMS.MT5 &&
+                    combined_cfd_mt5_account &&
+                    `${combined_cfd_mt5_account.sub_title}${short_code_and_region}`,
                 value: account.loginid,
```

Comment:
@thisyahlen-deriv we should leave the previous implementation
```
is_cfd && account.account_type === CFD_PLATFORMS.MT5 && combined_cfd_mt5_account
                        ? `${combined_cfd_mt5_account.sub_title}${short_code_and_region}`
                        : account_text_display,
```
because if our account is not MT5(Derived or Financial), but DXTRADE(Deriv X) or DERIVEZ(Deriv EZ) we should show proper text title, basically we use `${combined_cfd_mt5_account.sub_title}${short_code_and_region}` to show proper title for MT5(Derived or Financial) and `account_text_display` to show proper title for DXTRADE(Deriv X) and DERIVEZ(Deriv EZ).

Reply:
Alrights will replace